core-bundle/src/Command/UserCreateCommand.php

```diff
@@ -277,21 +277,28 @@ private function persistUser(string $username, string $name, string $email, stri
         $time = time();
         $hash = $this->encoderFactory->getEncoder(BackendUser::class)->encodePassword($password, null);
 
-        $this->connection->insert(
-            'tl_user',
-            [
-                'tstamp' => $time,
-                'name' => $name,
-                'email' => $email,
-                'username' => $username,
-                'password' => $hash,
-                'language' => $language,
-                'backendTheme' => 'flexible',
-                'admin' => $isAdmin,
-                'pwChange' => $pwChange,
-                'dateAdded' => $time,
-                'groups' => !$isAdmin && !empty($groups) ? serialize(array_map('strval', $groups)) : '',
-            ]
-        );
+        $data = [
+            'tstamp' => $time,
+            'name' => $name,
+            'email' => $email,
+            'username' => $username,
+            'password' => $hash,
+            'language' => $language,
+            'backendTheme' => 'flexible',
+            'admin' => $isAdmin,
+            'pwChange' => $pwChange,
+            'dateAdded' => $time,
+        ];
+
+        if (!$isAdmin && !empty($groups)) {
+            $data[$this->quote('groups')] = serialize(array_map('strval', $groups));
```

Comment:
```suggestion
            $data[$this->connection->quoteIdentifier('groups')] = serialize(array_map('strval', $groups));
```

No need to add a private method in this case.